### PR TITLE
Create Formulaic TOS

### PR DESCRIPTION
**Pull request by aroybalreid**
Terms of Service for Formulaic product.  The page should be listed here: https://www.mozilla.org/en-US/about/legal/, but the primary concern is having a URL that the Formulaic product page can link to.

### CODE REVIEW

File: Formulaic TOS [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+If there is a conflict or ambiguity between a translated version of these terms and the English language version, the English language version applies.
+
+##9. Contact Us
+Our [copyright or trademark claims reporting page](https://www.mozilla.org/about/legal/report-infringement/) explains how to report infringement on the Service.
+For other concerns, you may email us at [legal-notices@mozilla.com](mailto:legal-notices@mozilla.com) or write to us at the following address: 
+
+Mozilla Corporation
+Attn: Mozilla – Legal Notices
+149 New Montgomery Street
+4th Floor
```
Comment (peiying2) [inline Formulaic TOS:71]: should we update to `5th floor`

**Review by peiying2 — CHANGES_REQUESTED** [review]
Mostly formatting issues, need line breaks between paragraphs, heading and paragraphs; double space should be replaced with single space between sentences.

File: Formulaic TOS [hunk 2]
```diff
@@ -0,0 +1,73 @@
+#Terms of Service - Mozilla Formulaic
```
Comment (flodolo) [inline Formulaic TOS:1]:
```suggestion
# Mozilla Formulaic Terms of Service
```

There is a missing space, but also this is not the way other documents are titled.

File: Formulaic TOS [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+Last Updated March 8, 2024
+{: datetime="2024-03-08"
```
Comment (flodolo) [inline Formulaic TOS:4]:
```suggestion
{: datetime="2024-03-08" }
```

File: Formulaic TOS [hunk 4]
```diff
@@ -0,0 +1,73 @@
+#Terms of Service - Mozilla Formulaic
+
+Last Updated March 8, 2024
+{: datetime="2024-03-08" 
+
+Formulaic (the “Service”) is a service that allows users to create, share, manage, and use templates (the “Formulas”) for querying Artificial Intelligence (AI) based Large Language Model (LLMs) (the “Service”).  It also allows users to run those queries through a third party LLM platform, and to view the resulting responses. (For the purposes of this Agreement, Formulas, instructions, descriptions, and responses will all be referred to as the “Content”.) The Service is owned, licensed, and operated by Mozilla Corporation (referred to herein as "we," "us," "our," or "Mozilla"). 
+Please read these terms of service (“Agreement” or “Terms”) carefully before you sign up for or use the Service.  The Terms constitute a binding legal agreement, which govern your use of the Service via any platform or device.  By accessing, signing up for, or using the Service, you agree to be bound by these Terms.  
+
+##1. The Service
```
Comment (flodolo) [inline Formulaic TOS:9]: Missing space after the dash, missing empty line after a title (across the document).

File: Formulaic TOS [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+The Service (including both the platform and the Formulas shared by other users, as well as your ability to submit and/or run the Formulas using third party LLMs) is provided solely for use according to these Terms.  If you agree to these terms on behalf of a third party, such as your employer, you represent that you are authorized to, and intend to bind that party to these Terms.
+By accepting these terms, you also accept the Formulaic Privacy Notice and Formulaic Content Policies.
```
Comment (flodolo) [inline Formulaic TOS:11]: Line breaks don't work in markdown, unless you leave multiple spaces after the line ending before.

File: Formulaic TOS [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+###A. Creating an Account
+In order to view or run shared queries, or to share content of your own, you will need to create an account on the Service using your Google Account.  You can create a Google Account on [Google’s signup page](https://accounts.google.com/lifecycle/steps/signup/name?dsh=S1694799353:1707861924262237&flowEntry=SignUp&flowName=GlifWebSignIn&theme=glif&TL=ADg0xR0JpndiwBwSSLq5Ih_cBj1vPyVBRu2lVXVKxOqIHKvLx__ThXaRX7v0OVnt).  More information about the information that we receive from Google, and how we use that information, is included in the Formulaic Privacy Notice. Google Accounts has it’s own [Privacy Notice](https://policies.google.com/privacy?hl=en-US).
```
Comment (flodolo) [inline Formulaic TOS:14]: The link seems to include way too many parameters. Pointing to `https://accounts.google.com/` should be enough?

**Review by flodolo — COMMENTED** [review]
I've started reviewing the document, but there's a major problem that I've already pointed out in previous changes: the file name needs to be lowercase, and needs to include the `.md` extension.

Also, in this case, it needs to live in the `en`  folder, not in the root of the repository. I will make most changes directly, and leave more semantic one for later.

File: Formulaic TOS [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+Any ideas, suggestions, or feedback that you provide about the Service may be used by Mozilla for free, without any additional obligations.
+When you use the Service to run a query, the response will be provided by a third party LLM operator.  If you use the Google LLM, Google may use your (anonymized) content to train its AI model.  Outputs from LLMs may not always be accurate, and should not be relied on for factual information, or for advice related to finances, employment, education, housing, legal, medical, career, or other important personal decisions.  LLM outputs are affected by a number of factors such as inputs, training data, programming, and others, and may be incomplete, incorrect, or offensive, or refer to specific products, companies, or individuals.  They do not represent Mozilla’s views, or indicate an endorsement of any kind.
+
+###C. Term; Termination
```
Comment (flodolo) [inline Formulaic TOS]: What is "Term" in this context?

**Comment by flodolo** [comment]
Now that the file has the correct extension, you can preview and check if it looks as you expected (in particular line breaks)
https://github.com/mozilla/legal-docs/blob/6eae98ffc5091341797670bbd1370215b6fde44a/en/formulaic_tos.md

File: Formulaic TOS (hunk 1 above)
Comment (aroybalreid) [reply Formulaic TOS]: Yes please.  Thanks

File: Formulaic TOS (hunk 2 above)
Comment (aroybalreid) [reply Formulaic TOS]: Your suggestion looks good to me.

File: Formulaic TOS (hunk 6 above)
Comment (aroybalreid) [reply Formulaic TOS]: Yes, that works.  Thanks

File: Formulaic TOS (hunk 7 above)
Comment (aroybalreid) [reply Formulaic TOS]: Term = the term (length) of the contract.  The terms will continue to apply for an indefinite term until terminated by one of the parties

**Comment by aroybalreid** [comment]
Thanks for sharing the preview.  I see some spacing and other issues in there.  Can I make edits at this point in the process, or do I need to do a pull request after everything else is settled?

**Comment by flodolo** [comment]
Please make edits.

If you switch to the **Files changed** tab in this page, then you click on the 3 dots and select **View file** to check the latest version.


Screenshot 2024-03-19 alle 19 40 47


If you want to create a new paragraph, you need an empty line between the paragraphs. If you want just a new line, you need to leave two empty spaces at the end of the previous line.

**Comment by flodolo** [comment]
(to edit, from the 3 dots select **Edit file**)

**Comment by flodolo** [comment]
Also: when you're editing the file, at the top left you can switch to PREVIEW, then go back to editing.

**Review by peiying2 — APPROVED** [review]

**Review by flodolo — APPROVED** [review]

**Pull request merged**